cos_alerter/server.py

```diff
@@ -1,18 +1,13 @@
-import time
-import yaml
-
 from flask import Flask
 
 from .alerter import AlerterState
 
 app = Flask(__name__)
-with open('/etc/cos-alerter.yaml', 'rb') as f:
-    app.config.update(yaml.safe_load(f))
 
 
 @app.route('/alive', methods=['POST'])
 def alive():
     # TODO Decide if we should validate the request.
-    with AlerterState() as data:
-        data.alert_time = time.time()
+    with AlerterState() as state:
+        state.set_alert_time()
     return 'Success!'
```

Comment:
imho `reset_alert_timeout` would be a better name for this.

Reply:
what about `set_last_alert_time`?

Reply:
I agree with Pietro's suggestion